### ISSUE TITLE
How to Add More Epochs When the Model's Performance is Still Improving After 1000 Epochs

### ISSUE THREAD

**Issue by b-niu** [MIC-DKFZ/nnUNet#2652]
Dear authors,
I'm currently working with a model and I've encountered an interesting situation. After running 1000 epochs, I noticed that the performance of the model is still slowly increasing. Now I'm considering adding more epochs (for example, adding another 1000 epochs). I'm wondering which approach you would recommend:

1. Modify the default number of epochs to 2000 and train the model from scratch.
2. Modify the default number of epochs to 2000 and use the weights from the previous training session to perform the training.
3. Employ some specific techniques to continue the training for an additional 1000 times based on the previous weights.
4. Or is there any other better way that you could suggest?

I would really appreciate it if you could share your insights and recommendations on this matter. Thank you very much for your time and assistance.
Best regards.

**Comment by seziegler** [comment]
Hi @b-niu ,

due to the scheduled learning rate which is already very small in the end of the 1000 epochs it would be best to train from scratch for 2000 epochs.

**Comment by b-niu** [comment]
Thank you @seziegler !. I will close this issue.
